qcodes/instrument_drivers/AlazarTech/ATS.py

```diff
@@ -889,38 +889,23 @@ class AcquisitionController(Instrument):
         _alazar: a reference to the alazar instrument driver
```

Comment:
should be removed as the interface doesn't have this attribute

Reply:
One should not document private attributes anyway